Resources/Prototypes/_RMC14/Catalog/Fills/Crates/medical.yml

```diff
@@ -73,6 +73,7 @@
       - id: CMPillCanisterKelotane
       # - id: CMPillCanisterTramadol
       # - id: CMPillCanisterPeridaxon
+      - id: Bloodpack
```

Comment:
This needs to be an RMC bloodpack but I also don't think a single pack in this purchase is going to make a difference, maybe look into having a blood pack crate made instead? I'd need to see if something like that exists in 13...

Reply:
should i make it have multiple blood packs or make a new crate specifically for them?